cpp/daal/include/algorithms/k_nearest_neighbors/bf_knn_classification_model.h

```diff
@@ -52,6 +52,25 @@ enum DataUseInModel
     doNotUse = 0, /*!< The input data and labels will not be the component of the trained kNN model */
     doUse    = 1  /*!< The input data and labels will be the component of the trained kNN model */
 };
+/**
+ * <a name="DAAL-ENUM-ALGORITHMS__BF_KNN_CLASSIFICATION__RESULTTOCOMPUTEID"></a>
+ * Available identifiers to specify the result to compute
+ */
+enum ResultToComputeId
+{
+    computeClassLabels         = 0x00000001ULL,
```

Comment:
We have resultToEvaluate for that

Reply:
Removed. classLabels were solved through `none` value in ResultToComputeId of classifier